relay/sources/relayd/src/remote_run.rs

```diff
@@ -175,14 +176,13 @@ impl RunParameters {
         })
     }
 
-    pub fn command(&self, is_root: bool, test_mode: bool) -> (String, Vec<String>) {
+    pub fn command(&self, is_root: bool, test_mode: bool, job_config: Arc<JobConfig>) -> (PathBuf, Vec<String>) {
         let program = if test_mode {
-            "echo"
+            PathBuf::from("echo")
         } else {
-            // FIXME make it configurable
-            "/opt/rudder/bin/rudder"
-        }
-        .to_string();
+            let jobconfig2 = &job_config.clone();
+            PathBuf::from(jobconfig2.clone().cfg.remote_run.command.to_str().unwrap())
```

Comment:
`job_config.clone().cfg.remote_run.command.clone()` should be enough

Reply:
We have to clone because a reference is not enough as we have to return an owned `PathBuf`